hypha/apply/projects/models/report.py

```diff
@@ -182,19 +182,44 @@ class ReportConfig(models.Model):
 
     WEEK = _('week')
     MONTH = _('month')
+    YEAR = _('year')
+    ONE_TIME = _('one_time')
     FREQUENCY_CHOICES = [
+        ('', '---'),
+        (ONE_TIME, _('One Time')),
         (WEEK, _('Weeks')),
         (MONTH, _('Months')),
+        (YEAR, _('Years')),
```

Comment:
Also, the choices should be 

- Once
- Weekly
- Monthly
- Yearly

Reply:
Initially, I did change it to weekly, or yearly,... but I think I also need to understand the terminology here. We will discuss this in today's call.